rpcs3/Emu/Cell/Modules/cellRtc.cpp

```diff
@@ -571,20 +594,15 @@ error_code cellRtcParseRfc3339(vm::ptr<CellRtcTick> pUtc, vm::cptr<char> pszDate
 
 	vm::var<CellRtcDateTime> date_time;
 
-	// TODO fix unholyness
-
-	s64 l_add;
-	u32 pc_var_5;
-	u32 u_var_7;
-	s32 i_var_6;
-	s32 i_var_8;
+	const auto digit = [](char c) -> s32
+	{
+		return c - 48;
```

Comment:
```suggestion
		return c - '0';
```
I personally prefer this, but it should be clear in this context